lib/salemove/process_handler/notifier_factory.rb

```diff
@@ -4,6 +4,8 @@ class NotifierFactory
       def self.get_notifier(process_name, conf)
         if conf && conf[:type] == 'airbrake'
           AirbrakeNotifier.new
+        elsif conf && conf[:type] == 'sentry'
+          SentryNotifier.new
         elsif conf && conf[:type] == 'growl'
           GrowlNotifier.new(process_name)
         elsif conf && conf[:type] == 'terminal-notifier'
```

Comment:
At this point we might do `return unless conf` and then a switch over conf[:type].

Reply:
sure, changed